stable/ghost/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: ghost
-version: 3.0.2
-appVersion: 1.22.2
+version: 3.1.0
+appVersion: 1.22.1
```

Comment:
Why are you decreasing `appVersion`?

Reply:
appVersion in Chart.yaml did not match the actual image tag.  This change was to correct that.

Reply:
Happy to reverse if this is blocking the merge.

Reply:
@carrodher  ^

Reply:
thanks for fixing that @josdotso!